pymc/logprob/censoring.py

```diff
@@ -247,6 +248,8 @@ class MeasurableSwitchEncoding(MeasurableElemwise):
     """A placeholder used to specify the log-likelihood for a encoded RV sub-graph."""
 
     valid_scalar_types = (Switch,)
+    # number of measurable branches to facilitate correct logprob calculation
+    measurable_branches = 0
```

Comment:
This should be a property added during `__init__`. You must also add it as `__props__` so that PyTensor doesn't consider two Ops with the same inputs but different measurable_branches equivalent and tries to merge them.

Something like `__props__ = Elemwise.__props__ + ("measurable_branches",)`